### ISSUE TITLE
SSL Handshake Failed on iPad

### ISSUE THREAD

**Issue by tjolsen-vn** [ProxymanApp/Proxyman#844]
I can't see to proxy my apps successfully. Trying google as recommended by [the docs](https://docs.proxyman.io/troubleshooting/get-ssl-error-from-https-request-and-response), and that gets the `SSL Handshake Failed` error as well.

### Proxyman version? (Ex. Proxyman 1.4.3)
2.22.0

### macOS Version? (Ex. mac 10.14)
11.2.3

### Steps to reproduce 
- Follow instructions to install certificate on physical iOS device
- Try to proxy Google.com
- Get SSL Handshake Failed error
- Remove profile/certificate from iPad
- Turn off proxy on iPad
- Remove certificate from Mac (remove and reinstall also recommended by [docs](https://docs.proxyman.io/troubleshooting/get-ssl-error-from-https-request-and-response)
- Install new certificate on Mac
- Turn on proxy and install certificate on iPad again
- Try to proxy Google again
- Get SSL Handshake Failed error again

### Expected behavior
I've walked through the debugging steps, but can't seem to get even Google working. I would expect I can at least proxy Google, since that is the test you suggest.

**Comment by NghiaTranUIT** [comment]
@tjolsen-vn  If you could not intercept https://google.com from your iOS Device, it means the certificate is not set up properly.

From what you mention, you only "Install the certificate".

1. Can you verify that you've already trusted the certificate in your iOS Devices?

### Install Proxyman CA

iOS ≥ 10.3: Setting app > General > Profiles > Select Proxyman CA > Install
iOS ≥ 12.2: Setting app > Profiles Downloaded > Select Proxyman CA > Install
### Trust Proxyman CA

Setting app > General > About > Certificate Trust Settings > Switch ON on Proxyman CA.

<img width="669" alt="install_and_trust_proxyman_certificate" src="https://user-images.githubusercontent.com/5878421/113232596-adfb3200-92c7-11eb-845e-c3773f63b9c5.png">

2. Can you try to intercept https://google.com from Safari app in your macOS machine?
It would verify that you install and trust the certificate properly in your macOS too 👍

**Comment by tjolsen-vn** [comment]
Yeah simulator is fine and easy, but there are certain settings I can't test on a sim.

But yes I can confirm that I have trusted the certificate as well, should have made that more clear.

However, it seems to be working fine today. Not sure what was going on. Is it possible my network had something to do with it?

**Comment by NghiaTranUIT** [comment]
If it's a network, you might not see the traffic. But in your case, you can see it, but get SSL Error.

I suppose that might be it's from the device. If it happens in the future, please help me to reset the device. It might resolve your case.

Just wondering: What are your iOS version and device model?

**Comment by tjolsen-vn** [comment]
Yeah for sure if it happens again I'll try to get more details. Thanks for reaching out still.

I'm testing on an iPad (6th Gen) running iOS 14.4.1.

**Comment by tjolsen-vn** [comment]
For now I'd say this can be closed though. Getting everything I need today lol... now that I don't need it today haha. That's how it goes I guess. 😂 Maybe the Proxy Gods just really wanted me to brute force this without helpful tools yesterday.
